server/webdriver/tests/standalone/sdg_sanity.py

```diff
@@ -67,7 +67,7 @@ class Mode:
 OUTPUT_DIR = "output"
 
 COUNTRIES_JSON_FILE = "https://code.officialstatistics.org/undata2/data-commons/frontend-tester/-/raw/main/UNSD-Website-skeleton/UNSD.Website/ClientApp/src/templates/DataCommons/config/countries.json?ref_type=heads&inline=false"
-REGIONS_JSON_FILE = "sdg_regions.json"
+REGIONS_JSON_FILE = "https://code.officialstatistics.org/undata2/data-commons/frontend-tester/-/raw/main/UNSD-Website-skeleton/UNSD.Website/ClientApp/src/templates/DataCommons/config/geo_regions.json?ref_type=heads&inline=false"
```

Comment:
Also DC entities of type:  "UNGeoRegion", "ContinentalUnion", "Continent" ?  Can make a static list if you like.

Reply:
We had made a list earlier - does this one work? https://github.com/datacommonsorg/website/blob/1a941e6d1b81957003cbda6d3a6adf6d4cbf4e8e/server/webdriver/tests/standalone/sdg_regions.json

Reply:
Yes, if you've confirmed the content of `sdg_regions.json` is `geo_regions.json` + entities of those types?